frontends/ol-components/src/components/SimpleSelect/SimpleSelect.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react"
+import { Select } from "../SelectField/SelectField"
+import MenuItem from "@mui/material/MenuItem"
+import type { SelectChangeEvent } from "@mui/material/Select"
+
+interface SimpleSelectProps {
+  /**
+   * Value of initial selection for the dropdown
+   */
+  initialValue: string | string[]
+  /**
+   * Whether the dropdown allows multiple selections
+   */
+  isMultiple: boolean
+  /**
+  The function that runs when there is a selection from the dropdown
+   */
+  onChange: (event: SelectChangeEvent<string[] | string>) => void
+  /**
+   * The options for the dropdown
+   */
+  options: SimpleSelectOptionProps[]
+  /**
+   * Function that controls the display for the dropdown
+   */
+  renderValue?: (selected: string | string[] | void) => string
+  /**
+   * class name for the dropdown and base for key for dropdown options
+   */
+  className?: string
+}
+
+interface SimpleSelectOptionProps {
+  /**
+   * value for the dropdown option
+   */
+  key: string
+  /**
+   * label for the dropdown option
+   */
+  label: string
+}
+
+const SimpleSelect: React.FC<SimpleSelectProps> = ({
+  className,
+  initialValue,
+  isMultiple,
+  onChange,
+  options,
+  renderValue,
+}) => {
+  return (
+    <Select
+      multiple={isMultiple}
+      displayEmpty
+      value={initialValue}
+      onChange={onChange}
+      className={className}
+      renderValue={renderValue}
+    >
+      {options.map((option, index) => (
+        <MenuItem
+          value={option.key.toString()}
+          key={"option-".concat(index.toString())}
```

Comment:
Probably better to just use `option.value`. Keys should be unique and specific to the item. Index is usually a bad choice (and ESLint should warn about it, but probably won't if you prepend a string). Index is bad because the index isn't really unique to the item—item indices change if the list is reordered items are prepended.

It's probably not an issue here, since the menu items are unlikely to change, but IMO just use `option.value`. The values **need** to be unique anyway.